conf/messages

```diff
@@ -55,6 +55,17 @@ checkingRecords.ffid=Identifying file formats
 fileChecksResults.header=Record check results
 fileChecksResults.title=Record check results
 fileChecksResults.description=Here are your results
+forbiddenError.header=You are not permitted to see this page
+forbiddenError.body=This page is restricted to certain users only.
+internalServerError.header=Sorry, there is a problem with the service
+internalServerError.body=Try again later.
+unauthenticatedError.header=You must be logged in to see this page
+unauthenticatedError.body=Please login to continue.
+notFoundError.header=This page doesn't exist
+notFoundError.body1=If you typed a web address, check it is correct.
+notFoundError.body2=If you pasted the web address, check you copied the entire address.
+notFoundError.body3=If the web address is correct or you selected a link or button, contact The National Archives Help Desk to speak to someone about your transfers.
+notFoundError.body4=Alternatively, you can go back to start page
```

Comment:
The wording doesn't seem quite right here: "go back to start page". Also , should it be a link?

Reply:
Kirsten and Sarra mentioned that as a non-technical user if they were on this page, they wouldn't know how to go back to the home page, so having a link as such would make it a lot easier for the user to navigate. Also, I think the term 'go back' sounds a bit informal, so I have changed it to 'return to home page' as that feels more familiar to the users. What do you think?

Reply:
The new wording looks OK to me, thanks. I don't know if "return" or "go back" is preferred. I just meant that "go back to start page" wasn't grammatical. 🙂 

What about the link?

Reply:
No worries. 

For the link I included the `<a>` tags around the relevant text in the `messages` file, and then in the HTML file where I call the text, I used:  `@Html(Messages("notFoundError.body3"))` to ensure the link works. I know it may not be the best solution but this was something that seemed to work cleaner than any other options.

Reply:
Yeah, I'm actually not sure how you're supposed to handle things like links within the messages file. I think this is fine for now.

It looks like the link doesn't go anywhere, though, because it links to `#`. Where is the link supposed to go, and could you update it to go there?

Reply:
That link is supposed to go to the dashboard page, as if the user sees an error not found, they can return to the home page (which is our dashboard) where they can either start a new transfer/resume a transfer.